packages/terra-consumer-nav/src/components/nav-logo/NavLogo.scss

```diff
@@ -13,7 +13,7 @@
   }
 
   .img {
-    color: var(--terra-consumer-logo-text-color, #3d3d3d);
+    color: var(--terra-consumer-img-color, #3d3d3d);
```

Comment:
nit: I'd recommend `--terra-consumer-logo-img-color` to keep it specific to logo instead of general to img.